xml/book_solution_based.xml

```diff
@@ -39,8 +39,8 @@ HINT (toms) 2020-06-05:
     <xi:include href="common_copyright_gfdl.xml"/>
   </info>
 
-  <xi:include href="art_solution_automation.xml"/>
-  <xi:include href="art_solution_monitoring.xml"/>
+  <xi:include href="article_solution_automation.xml"/>
```

Comment:
same as above: should we rename to article_sap_automation.xml and  article_sap_monitoring.xml?